backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5436280_sys_09696_adding_accounts_in_deckungs.sql

```diff
@@ -1,4 +1,3 @@
-
 CREATE OR REPLACE VIEW report.Direct_Costing_ElementValue_selection AS
 SELECT 
 	e1.C_ElementValue_ID as lvl1_C_ElementValue_ID, e1.value as lvl1_value, e1.name as lvl1_name, ( e1.value || ' ' || e1.name ) as lvl1_label,
```

Comment:
looks like the change is missing in here :-)

Reply:
right; edited  the file..